interface/wx/webview.h

```diff
@@ -459,12 +459,50 @@ class wxWebView : public wxControl
     */
     virtual void Reload(wxWebViewReloadFlags flags = wxWEBVIEW_RELOAD_DEFAULT) = 0;
 
-    /**
-        Runs the given javascript code.
-        @note When using wxWEBVIEW_BACKEND_IE you must wait for the current
+	/**
+        Gets current IE emulation level for the current program.
+        @return Current wxWebViewIEEmulationLevel.
+        @note Only avaliable on wxWEBVIEW_BACKEND_IE.
```

Comment:
This implies that the function can't be called at all, whereas in reality, it can be, and just return `EMPTY`.

Something needs to be changed: either the implementation or the documentation.